pkg/codegen/docs/templates/resource.tmpl

```diff
@@ -86,15 +86,10 @@ The {{ .Header.Title }} resource accepts the following [input]({{ htmlSafe "{{<
 
 <!-- Output properties -->
 ### Outputs
-{{ if ne (len .OutputProperties) 0 }}
```

Comment:
Is there a reason this was removed?

Reply:
Yes. Since we're injecting an output property, there should always be at least one.

Reply:
Ah yes. That's what this PR does. Duh!

..carry on, sir :)